cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorMockTest.java

```diff
@@ -659,6 +665,161 @@ public void testKafkaScaleDown(Params params, VertxTestContext context) throws I
         });
     }
 
+    /** Test Jmxtrans on scale Up of kafka brokers */
+    @ParameterizedTest
+    @MethodSource("data")
+    public void testScaleUpKafkaBrokersOnJmx(Params params, VertxTestContext context) throws InterruptedException, ExecutionException, TimeoutException {
```

Comment:
TBH I'm a bit curious why did you added exactly these two tests. Why exactly do we need to test the JMXTrans with scale up / scale down? If you are just testing that the configmap contains the records for the new brokers, can't that be done in some easier tests? These tests are fairly expensive. (I'm not saying it is wrong, I just don't understand why it is right)

Reply:
True! I just thought that this would have been the best way of testing that (to test every combination) but I think that it is reasonable to just test the configmap data change.